src/Common/ThreadFuzzer.h

```diff
@@ -33,16 +33,14 @@ namespace DB
   *
   * Notes:
   * - it can be also implemented with instrumentation (example: LLVM Xray) instead of signals.
-  * - we should also make the sleep time random.
-  * - sleep and migration obviously helps, but the effect of yield is unclear.
```

Comment:
You accidentally removed this comment (second line).

Reply:
It was on purpose actually, because sched_yield should yield to the scheduler (based on some decision tree but whatever), so I wonder why this comment